packages/starknet-snap/src/getErc20TokenBalance.ts

```diff
@@ -35,7 +35,13 @@ export async function getErc20TokenBalance(params: ApiParams) {
 
     logger.log(`getErc20Balance:\nerc20Address: ${erc20Address}\nuserAddress: ${userAddress}`);
 
-    const resp = await callContract(network, erc20Address, 'balanceOf', [num.toBigInt(userAddress).toString(10)]);
+    const resp = await callContract(
+      network,
+      erc20Address,
+      'balanceOf',
+      [num.toBigInt(userAddress).toString(10)],
+      'pending',
```

Comment:
1. replace above function by `getBalance`
2. i feel like we can have a enum to store 'pending' & 'latest' (base on MM eslint rules, please name the constant to PascalCase)
3. getBalance is always using pending? or sometime latest depends on use case ?